src/types/streaming_server/device_info.rs

```diff
@@ -0,0 +1,7 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, PartialEq, Serialize, Deserialize, Debug)]
+#[serde(rename_all = "camelCase")]
+pub struct DeviceInfo {
```

Comment:
Would be nice to also include documentation.
I suppose this is information of the device running the  server?!